modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/internal/search/spi/model/permission/contributor/test/DepotEntrySearchPermissionFilterContributorTest.java

```diff
@@ -53,28 +53,30 @@ public class DepotEntrySearchPermissionFilterContributorTest {
 
 	@Test
 	public void testContribute() throws Exception {
+		User user = UserTestUtil.addUser();
+
+		long count = _getDepotEntrySearchCount(user);
+
 		_depotEntry = _depotEntryLocalService.addDepotEntry(
 			HashMapBuilder.put(
 				LocaleUtil.getDefault(), StringUtil.randomString()
 			).build(),
 			HashMapBuilder.put(
 				LocaleUtil.getDefault(), StringUtil.randomString()
 			).build(),
-			DepotConstants.TYPE_SPACE,
+			DepotConstants.TYPE_ANY,
```

Comment:
@marcogalluzzi `ANY` should not be used when creating depot entries, only when querying. Depot entries should be either spaces or asset libraries.

Reply:
You are right 🤦